main.go

```
@@ -2,21 +2,45 @@ package main
+		panic(err)
+	}
+	defer cacheClient.Close()
+
+	readWriteValkey(cacheClient)
+	parseFile("sample.csv")
+
+}
+
+func readWriteValkey(cacheClient cache.DistributedCache) {
+	ctx := context.Background()
+	cacheClient.Set(ctx, "Hello", "World")
```

golangci-lint (1.64.8)

Error return value of `cacheClient.Set` is not checked

(errcheck)